icu4c/source/i18n/double-conversion-bignum-dtoa.cpp

```diff
@@ -384,7 +384,7 @@ static void BignumToFixed(int requested_digits, int* decimal_point,
 // Returns an estimation of k such that 10^(k-1) <= v < 10^k where
 // v = f * 2^exponent and 2^52 <= f < 2^53.
 // v is hence a normalized double with the given exponent. The output is an
-// approximation for the exponent of the decimal approimation .digits * 10^k.
```

Comment:
The various `double-conversion*` files in the ICU repo actually come from an external project. The upstream repo is on GitHub here: https://github.com/google/double-conversion

So unfortunately these fixes will get *overwritten* the next time the double-conversion library is updated.

It looks like the typos exist upstream in the current version of the library:
https://github.com/google/double-conversion/blob/master/double-conversion/bignum-dtoa.cc#L373
https://github.com/google/double-conversion/blob/master/double-conversion/bignum-dtoa.cc#L551

Would you mind maybe filing an issue and/or creating a PR for the upstream repo? 
[It would be a nice way to contribute to the upstream project. :) ]

Reply:
Sure, I can do that in the upstream repo :) Should I return these words in these files to what they were?

Reply:
FYI I created a PR on the upstream repo :) I'll probably have to do several PR's there since there might be misspelled words I haven't got to yet, but I'll get to those as I'm going through the list in the ticket on ICU

Reply:
Update, it has been merged upstream :)

Reply:
Nice! -- Thank you!
(Btw, it might be a good idea to link to the upstream PR in the comment here, so that there's some cross-linking for the future in case we forget. :) )

Reply:
> Should I return these words in these files to what they were?

Since it's been fixed upstream I think it would be fine to fix this in the ICU repo now. When the double-conversion library is updated again then these will already be fixed and there won't be any diffs. :)

Reply:
of course :) the link is this:
https://github.com/google/double-conversion/pull/145

Reply:
Of course, here is the link! https://github.com/google/double-conversion/pull/145